portfolio/src/main/webapp/index.html

```diff
@@ -5,25 +5,58 @@
     <title>Jazmine's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    
+    <style type = "text/css">
```

Comment:
You can put all of this in `style.css` It's already linked in line 6 so it'll still work and keep all your CSS in the same place.